### PR TITLE
Models

### PR DESCRIPTION
**Pull request by JakeGloschat**
Creates Beer model.  Creates Network file for fetching random beer.  Creates helpers and extensions w/ a basic constants file and Network Errors.

### CODE REVIEW

File: NewBrewer/NewBrewer/Helper&Extensions/NetworkError.swift [hunk 1]
```diff
@@ -0,0 +1,37 @@
+//
+//  NetworkError.swift
+//  NewBrewer
+//
+//  Created by Jake Gloschat on 4/3/23.
+//
+
+import Foundation
+
+import Foundation
```
Comment (maxpoff) [inline NewBrewer/NewBrewer/Helper&Extensions/NetworkError.swift]: You have imported foundation twice, here. The dangers of using snippets 😉

**Review by maxpoff — APPROVED** [review]
Fix the double import, but otherwise looks good

File: NewBrewer/NewBrewer/Helper&Extensions/NetworkError.swift (hunk 1 above)
Comment (JakeGloschat) [reply NewBrewer/NewBrewer/Helper&Extensions/NetworkError.swift]: oh snap!  Thanks, max!

**Comment by JakeGloschat** [comment]
Fixed error pointed out by my reviewer, Max.

**Pull request merged**